docs/actionable-messages/index.md

```diff
@@ -102,11 +102,11 @@ Actionable messages are available to all customer mailboxes on Exchange Online i
 | Office 365 ProPlus Monthly Channel | Yes, in version 1705, Build 8201 | Yes, in version 1805, Build 9330 |
 | Office 365 ProPlus Semi-Annual Channel (Targeted) | Yes, in version 1708, Build 8431 | Coming Soon |
 | Office 365 ProPlus Semi-Annual Channel | Yes, in version 1708, Build 8431.2153 | Coming Soon |
-| Outlook 2016 for Mac | Coming Soon | Coming Soon |
+| Outlook for Mac | Coming Soon | Coming Soon |
```

Comment:
Please add 2016 back to these table entries. We need to be very clear on versions here.